### DISCUSSION TITLE
insight/faq/service-monitor/

### DISCUSSION THREAD

**Discussion by giscus** [DaoCloud/DaoCloud-docs#1459] [Announcements]
# insight/faq/service-monitor/

DaoCloud Enterprise 5.0 下一代容器化平台集大成者，引领云原生浪潮，推动数字化转型。

https://docs.daocloud.io/insight/faq/service-monitor/

**Comment by gssjl2008** [comment]
文档陈旧了，无法采集

**Reply by justedennnnn** [reply]
示例yaml 中缺少了 insight 需要的label，文档已更新：https://github.com/DaoCloud/DaoCloud-docs/pull/1657

**Comment by windsonsea** [comment]
关联 PR https://github.com/DaoCloud/DaoCloud-docs/pull/1657
